compiler/src/main/java/bullet/impl/ComponentProcessingStep.java

```diff
@@ -175,8 +207,7 @@ private String getName(TypeMirror type) {
           "$<}\n",
           method.type(), method.name(), method.kind() == ComponentMethodKind.PROVIDER_OR_LAZY ? ".get()" : "");
     }
-    // TODO: exception message
-    getBuilder.addCode("throw new $T();\n", IllegalArgumentException.class);
+    getBuilder.addCode("throw new $T(\"No get or Provides method found for \" + type.getName() + \" in " + elementName.simpleName() + "\");\n", IllegalArgumentException.class);
```

Comment:
Again an unrelated change.
I also disagree with the wording; "Provides methods" live in modules; I suppose you meant "methods that return a `Provider`".